From f6a4d082f58a9659f97e4511d25a26207b9386d2 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca.weiss@fairphone.com>
Date: Wed, 8 Nov 2023 19:00:26 +0200
Subject: [PATCH 4/4] Fix install path for pkgconfig files

These files don't belong into /usr/libexec/pkgconfig but into
/usr/lib/pkgconfig.
---
 src/common/public/CMakeLists.txt    | 2 +-
 src/downloads/client/CMakeLists.txt | 2 +-
 src/downloads/common/CMakeLists.txt | 2 +-
 src/uploads/common/CMakeLists.txt   | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/common/public/CMakeLists.txt b/src/common/public/CMakeLists.txt
index 8b98462c..4b1e8660 100644
--- a/src/common/public/CMakeLists.txt
+++ b/src/common/public/CMakeLists.txt
@@ -57,7 +57,7 @@ target_link_libraries(${TARGET}
 
 configure_file(${TARGET}.pc.in ${CMAKE_CURRENT_BINARY_DIR}/${TARGET}.pc @ONLY)
 
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${TARGET}.pc DESTINATION ${CMAKE_INSTALL_LIBEXECDIR}/pkgconfig)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${TARGET}.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 install(TARGETS ${TARGET} DESTINATION ${CMAKE_INSTALL_LIBDIR})
 install(FILES ${TRANSFERS_PUBLIC_HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/lomiri/transfers)
 install(FILES ${ERRORS_PUBLIC_HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/lomiri/transfers/errors)
diff --git a/src/downloads/client/CMakeLists.txt b/src/downloads/client/CMakeLists.txt
index 81f83e3a..d622434c 100644
--- a/src/downloads/client/CMakeLists.txt
+++ b/src/downloads/client/CMakeLists.txt
@@ -71,6 +71,6 @@ target_link_libraries(${TARGET}
 
 configure_file(${TARGET}.pc.in ${CMAKE_CURRENT_BINARY_DIR}/${TARGET}.pc @ONLY)
 
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${TARGET}.pc DESTINATION ${CMAKE_INSTALL_LIBEXECDIR}/pkgconfig)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${TARGET}.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 install(TARGETS ${TARGET} DESTINATION ${CMAKE_INSTALL_LIBDIR})
 install(FILES ${PUBLIC_HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/lomiri/download_manager)
diff --git a/src/downloads/common/CMakeLists.txt b/src/downloads/common/CMakeLists.txt
index 00ccf2b0..23fa1482 100644
--- a/src/downloads/common/CMakeLists.txt
+++ b/src/downloads/common/CMakeLists.txt
@@ -47,6 +47,6 @@ target_link_libraries(${TARGET}
 
 configure_file(${TARGET}.pc.in ${CMAKE_CURRENT_BINARY_DIR}/${TARGET}.pc @ONLY)
 
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${TARGET}.pc DESTINATION ${CMAKE_INSTALL_LIBEXECDIR}/pkgconfig)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${TARGET}.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 install(TARGETS ${TARGET} DESTINATION ${CMAKE_INSTALL_LIBDIR})
 install(FILES ${PUBLIC_HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/lomiri/download_manager)
diff --git a/src/uploads/common/CMakeLists.txt b/src/uploads/common/CMakeLists.txt
index d0849ee2..0729445f 100644
--- a/src/uploads/common/CMakeLists.txt
+++ b/src/uploads/common/CMakeLists.txt
@@ -41,6 +41,6 @@ target_link_libraries(${TARGET}
 
 configure_file(${TARGET}.pc.in ${CMAKE_CURRENT_BINARY_DIR}/${TARGET}.pc @ONLY)
 
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${TARGET}.pc DESTINATION ${CMAKE_INSTALL_LIBEXECDIR}/pkgconfig)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${TARGET}.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
 install(TARGETS ${TARGET} DESTINATION ${CMAKE_INSTALL_LIBDIR})
 install(FILES ${PUBLIC_HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/lomiri/upload_manager)
-- 
2.42.1

